exchange/utils_test.go

```diff
@@ -403,22 +403,6 @@ func TestCreateSanitizedImpExt(t *testing.T) {
 			},
 			expectedError: "",
 		},
-		{
-			description: "Marshal Error - imp.ext.prebid",
```

Comment:
Why did this test case go away?

Reply:
Same reason as above (https://github.com/prebid/prebid-server/pull/3225#discussion_r1362466347)